p2p/src/peer_manager/mod.rs

```diff
@@ -808,14 +808,10 @@ where
                     peer.ping_last = Some(ping_time_last);
                     peer.ping_min = Some(ping_time_min);
                 } else {
-                    log::debug!(
-                        "wrong nonce in ping response: {}, expected: {}",
-                        nonce,
-                        sent_ping.nonce
-                    );
+                    log::debug!("wrong nonce in ping response from peer {}", peer_id,);
```

Comment:
I suggest keeping the nonce, and just moving the nonce value after the text you're searching for. This way the assert passes in the tests and you can still have the extra information in the logs.